builtin/provisioners/file/resource_provisioner.go

```diff
@@ -77,6 +78,21 @@ func applyFn(ctx context.Context) error {
 	}
 }
 
+// Validate checks if the required arguments are configured
+func Validate(d *schema.ResourceData) (ws []string, es []error) {
+	numSrc := 0
+	if _, ok := d.GetOk("source"); ok == true {
+		numSrc++
+	}
+	if _, ok := d.GetOk("content"); ok == true {
```

Comment:
it seems [GetOk](https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_data.go#L89) fail to return `ok` in the following part:
https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_data.go#L91

The struct is:
```
r: {${data.template_file.config.rendered} <nil> true true 0xc4203c12c0}
```

Which means `r.Exists` and `r.Computed` but expecting `r.Exists && !r.Computed`
I was able to make a workaround with following code:
```
func validateFn(d *schema.ResourceData) (ws []string, es []error) {
	numSrc := 0
	content := d.Get("content").(string)
	source := d.Get("source").(string)

	if content != "" {
		numSrc++
	}
	if source != "" {
		numSrc++
	}

	if numSrc != 1 {
		es = append(es, fmt.Errorf("Must provide one of 'content' or 'source'"))
	}
	log.Printf("Numsrc is: %v", numSrc)
	return
}
```